build.gradle

```diff
@@ -169,6 +169,19 @@ project('shared:authplugin') {
 }
 
 project('shared:security') {
+    task jarTest (type: Jar) {
```

Comment:
Can you explain why these changes are needed in the PR `description > what the code does?`?

Reply:
many packages had test dependency on the ```shared: security``` i.e the tests in many packages depended on tests of ```shared: security``` and the best way of including such dependency is to create a jarTest for ```shared:security``` jarTest task builds the jar from the test outputs adding a ‘test’ to the jar name then I have created a configuration which basically is a group of dependencies and finally added the jar artifact to the configuration I have passed the configuration as parameter to the project that depends on it , thus whenever someone(project which is dependent on it) ask for testOutput, gradle will run the jarTest task